contents.json

```diff
@@ -60,8 +60,8 @@
     "title": "Converters",
     "id": "converters"
   }, {
-    "title": "Demo Apps",
-    "id": "demo-apps",
+    "title": "Other Awesome Lists",
+    "id": "other-awesome-lists",
```

Comment:
@serhii-londar Hi there! I think it would be better to replace the `category` value to `other-awesome-lists` for these items:
``` 
{
  "title": "open-source-ios-apps",
  "category": "demo-apps",
  "description": "A collaborative list of open-source iOS Apps.",
  "homepage": "https://github.com/dkhamsing/open-source-ios-apps"
}
{
  "title": "awesome-macOS",
  "category": "demo-apps",
  "description": "A curated list of awesome applications, softwares, tools and shiny things for macOS.",
  "homepage": "https://github.com/iCHAIT/awesome-macOS"
}
``` 
Otherwise, we can't see them 😃.

Reply:
@vpeschenkov Done.